tools/gke/kubernetes_api.py

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env python2.7
```

Comment:
have you looked at python packages like https://pypi.python.org/pypi/python-kubernetes/0.2? perhaps they provide the functionality you need.